op-batcher/readme.md

```diff
@@ -60,15 +60,16 @@ The `receiptsLoop` which
 
 The `throttlingLoop` which
 
-1. Waits for a signal either from the `blockLoadingLoop` or from the active sequencer endpoint provider (if more than one sequencer endpoint is provided)
-2. If it detects that the pending data in state is over a threshold, calls the sequencer over RPC and tells it to throttle the amount of L2 data it produces. See the (section below)[#data-availability-backlog]
+1. Waits for a signal from the `blockLoadingLoop`
+2. If it detects that the pending data in state is over a threshold, calls (potentially multiple) sequencers over RPC and tells them to throttle the amount of L2 data they produce. See the (section below)[#data-availability-backlog]. Each sequencer is throttled in parallel by a separate sub-goroutine (not shown in the below diagram).
```

Comment:
What about unthrottling actually? Don't we also call with `0`s to unthrottle?

Reply:
Should we also mention that additional throttling endpoints can be configured, e.g. for throttling builders in a rollup boost setup?

Reply:
[ce8a1cc](https://github.com/ethereum-optimism/optimism/pull/15603/commits/ce8a1cc4d0e114002d251bf410c68ca53005d38d)